commands/build/lib/build.js

```diff
@@ -135,6 +135,8 @@ const config = ({
         }),
         nodeResolve({
           extensions,
+          browser,
+          preferBuiltins,
```

Comment:
I see that the default value for `preferBuiltins` is `true` for `@rollup/plugin-node-resolve`. So having it default to `false` in nebula build would alter todays behaviour.

I suggest you change it's default value `true` to avoid changing the default build behaviour.